cmd/kubelet/app/options/container_runtime.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	// When these values are updated, also update test/e2e/framework/util.go
```

Comment:
update the comment to point to the right manifest.go file

Reply:
done